### PR TITLE
refactor: use `@blocksuite/icons`

### PR DESCRIPTION
**Pull request by Himself65**

### CODE REVIEW

File: packages/global/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     ]
   },
   "dependencies": {
+    "@blocksuite/icons": "=2.1.0-alpha.7",
```
Comment (doodlewind) [inline packages/global/package.json:38]: Where is the upstream repository? I am concerned if we hold private dependencies.

**Review by doodlewind — CHANGES_REQUESTED** [review]
We should not hold this private dependency, unless we open source the icon repo and make it publicly accessible.

File: packages/global/package.json (hunk 1 above)
Comment (himself65) [reply packages/global/package.json:38]: https://github.com/toeverything/icons

**Comment by doodlewind** [comment]
Opinionated closing this since we don't plan to make to icon workflow public recently.

**Pull request closed without merging**